apps/twap/src/migrations/1688633087604-migration.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Migration1688633087604 implements MigrationInterface {
+  name = 'Migration1688633087604';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            ALTER TABLE "order" DROP COLUMN "chainId"
+        `);
+    await queryRunner.query(`
+        TRUNCATE TABLE "order"
+        `);
```

Comment:
Does it need to truncate the whole table?

Anyway, doesn't actually matter as this is not production ready yet.

Disregard

Reply:
Nitpick:

```suggestion
    await queryRunner.query(`
            TRUNCATE TABLE "order"
        `);
```